tools/integration_tests/readonly/readonly_test.go

```diff
@@ -115,18 +115,17 @@ func TestMain(m *testing.M) {
 
 	var err error
 	ctx = context.Background()
-	var cancel context.CancelFunc
-
-	ctx, cancel = context.WithTimeout(ctx, time.Minute*20)
 	storageClient, err = client.CreateStorageClient(ctx)
```

Comment:
We can define cacheDir := "cache-dir-readonly-hns-"+ setup.IsHierarchicalBucket(ctx, storageClient) in one line avoiding repetitive Ifs in multiple tests.

Reply:
Done